documentation/intro_quick_start.md

```diff
@@ -26,42 +27,42 @@ After logging in, you will see a Resource Dashboard.
 You can learn more about it in the {ref}`forsc-dashboard` section.
 For now, go to the left-hand menu and choose JupyterHub under Compute.
 
-## 4. Select a Server Type
+## 4. Select a Server Type and Environment
 
-You’ll be prompted to select the size of the compute instance.
-Please choose **Small** until you have carefully read the {ref}`intro-best-practices` section.
+You’ll be prompted to select the following:
 
-## 5. Select a Software Environment
+-   **Server Type**: Size of the compute instance.
+    Please choose **Small** until you have carefully read the {ref}`intro-best-practices` section.
+-   **Environment**: Container image providing your base environment.
+    The **Default Astrophysics** environment is recommended for most use cases.
+    {ref}`base-environment` describes the options in more detail.
 
-Choose the **Default Astrophysics** environment unless you know that you have a particular need for one of the others.
+## 5. Launch JupyterLab
 
-## 6. Launch JupyterLab
```

Comment:
This is all OK now, but I wonder if we should leave the numbering up to the rendering job instead?

Reply:
That could be nice. I played around with it and got it working on the actual page but couldn't get the TOC to also show the numbering.

Reply:
Oh, but I considered the toc not showing the numbers a feature 😅